Tensile/KernelWriter.py

```diff
@@ -1543,6 +1547,15 @@ def noLoadLoop( self, kernel, tensorParametersA, tensorParametersB, isOptNLL, is
     # the scheduled GlobalRead,Inc code of PAP is inside openSumAtLeastUnroll (if PAP=on)
     kl.append(self.openSumAtLeastUnroll(kernel, prefetch=False, isOptNLL=isOptNLL, isPap=isPap))
 
+    if self.prefetchAcrossPersistent and kernel["PrefetchAcrossPersistentMode"] == 1 and isPap and not isOptNLL:
+      kStr = ""
+      kStr += str(self.openPrefetchAcrossPersistent(kernel, isOptNLL=False, useBufferOOB=True))
+      newTileCodes = self.setupNewTile(kernel, self.tPA, self.tPB, isPap=True, isOptNLL=True)
```

Comment:
Is isOptNLL=True correct here?
If it is intended, it might be good to have some comment here.